curp/src/server/raw_curp/tests.rs

```diff
@@ -38,6 +39,14 @@ impl<C: 'static + Command> RawCurp<C> {
             .iter()
             .map(|id| (id.clone(), Arc::new(Event::new())))
             .collect();
+
+        let role_change = {
+            let mut role_change = MockRoleChange::default();
+            role_change.expect_on_election_win().returning(|| {});
```

Comment:
Shall we add some notifier or channel inside the closure so that we can get informed when it's called?

Reply:
At the moment we don't have such a requirement and we can wait until we really need it to add it.